filecoin-proofs/src/types/mod.rs

```diff
@@ -108,6 +108,18 @@ pub struct SectorUpdateProofInputs {
     pub comm_d_new: Commitment,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct SealRegressionRecord {
+    pub porep_config: PoRepConfig,
+    pub comm_r: Commitment,
+    pub comm_d: Commitment,
+    pub prover_id: ProverId,
+    pub sector_id: SectorId,
+    pub ticket: [u8; 32],
+    pub seed: [u8; 32],
+    pub proof: Vec<u8>,
+}
+
```

Comment:
If this type is put into the `filecoin-proofs/tests/regression.rs` file it could be private and hence wouldn't be part of the public interface.